src/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = (props) => {
+  let { result } = props;
+  if (result === undefined) {
+    result = '0';
+  }
+  return (
+    <div>
+      <input value={result} style={{ textAlign: 'right' }} readOnly />
+    </div>
+  );
+};
+// Display.defaultProps = {
+//   result: '0',
```

Comment:
hey, this is triggering linting errors because you set it in propTypes as required. default values are set for props that may or may not be there. So let's set  the default so there is '0' showing before calculation.
- you can still set the propType but let it not be required, like so:
```
Display.propTypes = {
  result: PropTypes.string,
};
```

Reply:
Yes, that's what I'm doing just before the return statement.